web/test-utils/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ kotlin {
         val jsMain by getting {
             dependencies {
                 implementation(project(":internal-web-core-runtime"))
+                implementation(project(":web-core"))
```

Comment:
This is the only part I have concerns about. Technically we need this dependency because of a single (crucial one but nevertheless) function.

Reply:
What is this function?

Reply:
@olonho it's about renderComposable - it's an important function though - otherwise we were testing in unit tests equivalent but separate construct.